content/platforms/kubernetes/reference/README.md

```diff
@@ -0,0 +1,107 @@
+---
```

Comment:
@alexmilowski @bmansheim This also seems like a Task section doc. And it needs a concept article as well and maybe an optional getting started section for the installation. WDYT?

Reply:
Similarly, we should not include this in this PR. This is not reference but a task/how-to guide. This content also should be refactored and not called "README"

Reply:
We should not include this document in this PR. It probably belongs in the tasks section and also overlaps with the GESHER.md document.